docs/sources/flow/install/linux.md

```diff
@@ -9,35 +9,64 @@ You can install Grafana Agent Flow as a systemd service on Linux.
 
 ## Install on Debian or Ubuntu
 
-To install Grafana Agent Flow on Debian or Ubuntu, complete the following steps:
+To install Grafana Agent Flow on Debian or Ubuntu, run the following commands in a terminal window.
 
-1. Open a terminal and run the following command to install Grafana's package repository:
+1. Import the GPG key and add the Grafana package repository:
 
    ```shell
     sudo mkdir -p /etc/apt/keyrings/
     wget -q -O - https://apt.grafana.com/gpg.key | gpg --dearmor | sudo tee /etc/apt/keyrings/grafana.gpg > /dev/null
     echo "deb [signed-by=/etc/apt/keyrings/grafana.gpg] https://apt.grafana.com stable main" | sudo tee /etc/apt/sources.list.d/grafana.list
    ```
 
-2. Update the repositories:
+1. Update the repositories:
 
    ```shell
    sudo apt-get update
    ```
 
-3. Install Grafana Agent Flow:
+1. Install Grafana Agent Flow:
 
    ```shell
    sudo apt-get install grafana-agent-flow
    ```
 
-## Install on RedHat, RHEL, or Fedora
+### Uninstall on Debian or Ubuntu
 
-To install Grafana Agent Flow on RedHat, RHEL, or Fedora, complete the following steps:
+To uninstall Grafana Agent Flow on Debian or Ubuntu, run the following commands in a terminal window.
 
-1. Create `/etc/yum.repos.d/grafana.repo` with the following content:
+1. Stop the systemd service for Grafana Agent Flow:
+   
+   ```shell
+   sudo systemctl stop grafana-agent-flow
+   ```
+
+1. Uninstall Grafana Agent Flow:
 
+   ```shell
+   sudo apt remove grafana-agent-flow
    ```
+
+1. Optional. Remove the Grafana repository:
```

Comment:
The recommended syntax for [optionals](https://developers.google.com/style/procedures#optional-steps) is to use a colon following the optional indicator: 

```suggestion
1. Optional: Remove the Grafana repository:
```

Ditto in all the other places optional is used in this change

Reply:
Should we note that this should only be done if no other packages are installed from that repository?

Reply:
Stylistically, I have use `(Optional) Remove the Grafana repository` with no colon. This is something I can bring to the style meeting with the Docs Team. (Please use my version for now.)

Reply:
Good point. I hadn't got around to the final style pass :-) A quick grep of the Grafana/Grafana and Agent repos... and `(Optional)` is the one we are using everywhere vs. colons.